media-libs/mesa/mesa-9999.ebuild

```diff
@@ -16,6 +16,7 @@ HOMEPAGE="https://www.mesa3d.org/ https://mesa.freedesktop.org/"
 
 if [[ ${PV} == 9999 ]]; then
 	EGIT_REPO_URI="https://gitlab.freedesktop.org/mesa/mesa.git"
+	EGIT_BRANCH="main"
```

Comment:
Why would this be needed?
`git-r3.eclass` does not hardcode branch name `master`.

Reply:
Indeed, I don't define EGIT_BRANCH and use git-r3 on a few packages with "main" and there's no issues as-is.

Reply:
Ah I thought it was hard coded, but I can see it uses HEAD